### PR TITLE
Enhanced styling and routes

### PR DESCRIPTION
**Pull request by codegracie**
Added styling, routes and two new layouts.

### CODE REVIEW

**Comment by kian1213** [comment]
Ok

File: app/views/categories/index.html.erb [hunk 1]
```diff
@@ -3,5 +3,9 @@
     <%= t ".panel_title".html_safe %>
   </div>
   <div class="panel-body">
+
+	
+		
+
```
Comment (MuhammadTamzid) [inline app/views/categories/index.html.erb]: Remove unused space

File: app/views/home/index.html.erb [hunk 2]
```diff
@@ -17,9 +17,33 @@
       </div>
       <div class="col-md-8">
         <div class="row">
-          <%= link_to "#{t '.button_words'}".html_safe, words_path, class: "btn btn-primary btn-lg" %>
-          <%= link_to "#{t '.button_lessons'}".html_safe, categories_path, class: "btn btn-primary btn-lg" %>
+
+
+    
+
```
Comment (MuhammadTamzid) [inline app/views/home/index.html.erb]: Remove unused space

File: app/views/home/index.html.erb [hunk 3]
```diff
@@ -17,9 +17,33 @@
...
+      <p>
+        <%= link_to "#{t '.button_lessons'}".html_safe, categories_path, class: "btn btn-primary btn-lg" %>
+      </p> 
+      </div> </div> </div>
+
+
+      
+
+
+
```
Comment (MuhammadTamzid) [inline app/views/home/index.html.erb]: Remove unused space

File: app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -1,14 +1,19 @@
 <!DOCTYPE html>
+
 <html>
 <head>
-  <title><%= t ".title".html_safe %></title>
-  <%= stylesheet_link_tag    "application", media: "all", "data-turbolinks-track": true %>
-  <%= javascript_include_tag "application", "data-turbolinks-track": true %>
+  <title><%= t '.title'.html_safe %></title>
+  <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>
```
Comment (MuhammadTamzid) [inline app/views/layouts/application.html.erb]: double quote & convention

File: app/views/layouts/application.html.erb [hunk 5]
```diff
@@ -1,14 +1,19 @@
...
+  <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
```
Comment (MuhammadTamzid) [inline app/views/layouts/application.html.erb]: double quote & convention

File: config/routes.rb [hunk 6]
```diff
@@ -3,7 +3,13 @@
...
   # You can have the root of your site routed with "root"
-  root "home#index"
+  #root "home#index"
+
+  root "home#signup"
+
+  get "home" => "home#index", as: "home"
+  get "about" => "home#about", as: "about"
+
   get "words" => "words#index", as: "words"
```
Comment (MuhammadTamzid) [inline config/routes.rb]: Use resouces

File: config/routes.rb [hunk 7]
```diff
@@ -3,7 +3,13 @@
...
   get "categories" => "categories#index", as: "categories"
```
Comment (MuhammadTamzid) [inline config/routes.rb]: Use resouces

**Pull request closed without merging**